src/state/actions/user.js

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable camelcase */
+/* eslint-disable import/prefer-default-export */
+import axios from 'axios';
+import * as types from '../types';
+
+export const editProfile = (value, id) => async dispatch => {
+  dispatch({
+    type: types.EDIT_PROFILE_PICTURE,
+  });
+  try {
+    const { data } = await axios.patch(
+      `https://lambdadoor-staging.herokuapp.com/users/${id}`,
+      value
+    );
+    dispatch({
+      type: types.EDIT_PROFILE_PICTURE_SUCCESS,
+      payload: data[0],
+    });
+  } catch (error) {
+    dispatch({
+      type: types.EDIT_PROFILE_PICTURE_FAILURE,
+      payload: error.message,
+    });
+  }
+};
```

Comment:
We need to have different actions and types for the different profile edits. It'll be useful when we want to implement error handling